resources/UserActions.ts

```diff
@@ -0,0 +1,42 @@
+import { UserData } from './UserData';
+const bcrypt = require('bcrypt');
+
+export class UserActions extends UserData {
+    private _PASSWORD_HASH_SALT = 10;
+    
+    //just in case we need the salt value somewhere else in the future
+    public get salt() {
+        return this._PASSWORD_HASH_SALT;
+    }
+
+    public findUserByEmail(email: string) {
```

Comment:
Do you consider this method to be a user action?

Reply:
Definitely isn't. It has been moved.